### DISCUSSION TITLE
How to create an environmental variable containing other one?

### DISCUSSION THREAD

**Discussion by konradkalemba** [vercel/vercel#5215] [Help]
Hello, as the title states - I want to use env variable in other one, such as:
![image](https://user-images.githubusercontent.com/8682104/93880107-35466800-fcdd-11ea-9d42-8f120559ad30.png)

Is it possible in Vercel?

**Comment by mcsdevv** [comment]
Hello @konradkalemba, this isn't something that's currently possible. Could you explain your use case?
